src/components/DatePicker.js

```diff
@@ -1,11 +1,10 @@
 import React, { Component } from 'react';
 import { bindActionCreators } from 'redux';
-import { connect } from 'react-redux'
-import { withRouter, Redirect } from 'react-router-dom'
-import { Navbar, Nav, NavItem, Breadcrumb } from 'react-bootstrap';
-import _ from 'lodash'
```

Comment:
Here is an important point for the later production step:
lodash is a quite heavy library with a big number of methods that actually we don't use.

better is to define in the package all the micro libs of lodash we are using https://www.npmjs.com/browse/keyword/lodash-modularized

and in the js:
```
import merge from 'lodash.merge'
```
for example.

Reply:
#37